DataHandler.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+
+class DataHandler():
+
+    def __init__(self, service):
+        self.service = service
+
+    def verify(self, data):
+        return None
+
+    def return_data(self, response):
+
+        # Process daily data. The order of variables needs to be the same as requested.
+        daily = response.Daily()
+        daily_temperature_2m_max = daily.Variables(0).ValuesAsNumpy()
+        daily_temperature_2m_min = daily.Variables(1).ValuesAsNumpy()
+
+        daily_data = {"date": pd.date_range(
+            start = pd.to_datetime(daily.Time(), unit = "s", utc = True),
+            end = pd.to_datetime(daily.TimeEnd(), unit = "s", utc = True),
+            freq = pd.Timedelta(seconds = daily.Interval()),
+            inclusive = "left"
+        )}
+        daily_data["temperature_2m_max"] = daily_temperature_2m_max
+        daily_data["temperature_2m_min"] = daily_temperature_2m_min
+
+        daily_dataframe = pd.DataFrame(data = daily_data)
```

Comment:
the data handler should generally not generate its own data. you should invoke your passed in service's `return_data` method